### PR TITLE
different  sysNavbar for osx and win

### PR DESCRIPTION
**Pull request by Ladvace**

### CODE REVIEW

File: src/app/desktop/components/SystemNavbar.js [hunk 1]
```diff
@@ -24,38 +27,92 @@ const SystemNavbar = () => {
...
+  width: 100%;
+  height: ${({ theme }) => theme.sizes.height.systemNavbar}px;
+  background: ${({ theme }) => theme.palette.grey[900]};
+  -webkit-app-region: drag;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
   & > * {
     -webkit-app-region: no-drag;
     width: 35px;
```
Comment (blarfoon) [inline src/app/desktop/components/SystemNavbar.js]: Too much duplicated code here. Please extract the logic and reduce code complexity

File: src/app/desktop/components/SystemNavbar.js [hunk 2]
```diff
@@ -6,9 +6,12 @@ import {
   faWindowMinimize,
   faWindowMaximize,
   faWindowRestore,
-  faTimes
+  faTimes,
+  faTerminal
 } from '@fortawesome/free-solid-svg-icons';
 
+const osx = process.platform === 'darwin';
```
Comment (blarfoon) [inline src/app/desktop/components/SystemNavbar.js]: Variable should be called `isOsx` since it's a condition

File: public/electron.js [hunk 3]
```diff
@@ -195,6 +195,11 @@ ipcMain.handle('openFolder', (e, folderPath) => {
   shell.openItem(folderPath);
 });
 
+ipcMain.handle('open-devTools', () => {
+  console.log('CIAO');
```
Comment (blarfoon) [inline public/electron.js]: Remove useless console log

File: public/electron.js [hunk 4]
```diff
@@ -195,6 +195,11 @@ ipcMain.handle('openFolder', (e, folderPath) => {
   shell.openItem(folderPath);
 });
 
+ipcMain.handle('open-devTools', () => {
```
Comment (blarfoon) [inline public/electron.js]: This should all be lowercase

File: src/app/desktop/components/SystemNavbar.js [hunk 5]
```diff
@@ -24,38 +27,92 @@ const SystemNavbar = () => {
...
-        css={`
-          font-size: 18px;
-        `}
-        onClick={() => ipcRenderer.invoke('quit-app')}
-      >
-        <FontAwesomeIcon icon={faTimes} />
-      </div>
-    </Container>
+  const openDevTools = () => {
+    ipcRenderer.invoke('open-devTools');
```
Comment (blarfoon) [inline src/app/desktop/components/SystemNavbar.js]: This should all be lowercase

**Review by blarfoon — CHANGES_REQUESTED** [review]

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- src/app/desktop/components/SystemNavbar.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5243673&bid=17187152)

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- src/app/desktop/components/SystemNavbar.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5243673&bid=17187152)

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- src/app/desktop/components/SystemNavbar.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5243673&bid=17187152)

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- src/app/desktop/components/SystemNavbar.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5243673&bid=17187152)

**Comment by blarfoon** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- src/app/desktop/components/SystemNavbar.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5243673&bid=17187152)

**Pull request merged**